identity/aziot-identityd/src/identity.rs

```diff
@@ -553,12 +553,14 @@ impl IdentityManager {
                         identity_cert,
                         identity_pk,
                     } => {
-                        self.create_identity_cert_if_not_exist_or_expired(
-                            &identity_pk,
-                            &identity_cert,
-                            &device_id,
-                        )
-                        .await?;
+                        //IoT Hub doesn't require device ID to match identity certificate CN
```

Comment:
Nit: Should be space after `//`. Same with line 619 below.

Reply:
Fixed here and below.